docs/exploration-and-organization/delete-and-restore.md

```diff
@@ -0,0 +1,122 @@
+
+To restore an item:
+
+1. Go to Trash;
+2. Find the item you'd like to delete. You can sort deleted items to make it easier to find the item, or [search for your question in Trash](#search-in-trash);
+3. Click on the checkbox next to the item to select it;
+4. Select "Restore".
+
+> Restoring a collection will also restore all the items from that collection.
+
+If the item's original parent collection has been deleted as well, you won't see an option to **Restore**. You'll still be able to move the it from Trash to a different collection.
+
+## How deleting an item affects related items
+
+Deleting or restoring an item will affect other items that depend on that item.
+
+### Questions
+
+| Affected item                      | In Trash       | Permanently deleted                | Restored       |
+| ---------------------------------- | -------------- | ---------------------------------- | -------------- |
+| Dashboard                          | Card removed   | Card removed                       | Card restored  |
+| Question based on deleted question | Works normally | Breaks with `Card not found` error | Works normally |
+| Alerts                             | Removed        | Removed                            | Not restored   |
+
+### Dashboards
+
+| Affected item   | In Trash                   | Permanently deleted        | Restored |
+| --------------- | -------------------------- | -------------------------- | -------- |
+| Subscriptions   | Deactivated                | Deactivated                | Restored |
+| Custom homepage | Revert to default homepage | Revert to default homepage | Restored |
+
+### Model
+
+| Affected item                      | In Trash       | Permanently deleted                | Restored       |
+| ---------------------------------- | -------------- | ---------------------------------- | -------------- |
+| Question based on deleted question | Works normally | Breaks with `Card not found` error | Reactivated    |
+| Dashboard                          | Card removed   | Card removed                       | Card restored  |
+| Action                             | Works normally | Deleted                            | Works normally |
+
+## Collections
+
+| Affected item                                  | In Trash | Permanently deleted | Restored |
+| ---------------------------------------------- | -------- | ------------------- | -------- |
+| All items and subcollecitons in the collection | In Trash | Permanently deleted | Restored |
+
+## Permanently deleting items
+
+Moving an item to Trash is not permanent: you'll be able to restore it to the original parent collection, or move it to a non-deleted collection.
+
+To permanently delete an item:
+
+1. Go to Trash;
+2. Find the item you'd like to delete;
+3. Click on the checkbox next to the collection to select it;
+4. Select "Permanently delete".
```

Comment:
Maybe some warning about how you won't be able to recover the item. It's lost to the void.